src/cockpit/internal_endpoints.py

```diff
@@ -0,0 +1,138 @@
+# This file is part of Cockpit.
+#
+# Copyright (C) 2022 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import grp
+import logging
+import os
+import pwd
+
+from systemd_ctypes import bus
+
+logger = logging.getLogger(__name__)
+
+
+class ConfigEndpoint(bus.Object):
+    def __init__(self):
+        ...
+
+    @bus.Object.method(out_types='u', in_types='suuu')
+    def get_u_int(self, name, _minimum, default, _maximum):
+        return default
+
+
+@bus.Object.interface('cockpit.LoginMessages')
+class LoginMessagesEndpoint(bus.Object):
+    ...
+
+
+@bus.Object.interface('cockpit.Machines')
+class MachinesEndpoint(bus.Object):
+    @bus.Object.method(in_types=['s', 's', 'a{sv}'])
+    def update(self, *args):
+        ...
+
+    @bus.Object.property('a{sa{sv}}')
+    def machines(self):
+        return {}
+
+
+@bus.Object.interface('cockpit.Packages')
+class PackagesEndpoint(bus.Object):
+    ...
+
+
+@bus.Object.interface('cockpit.Superuser')
+class SuperuserEndpoint(bus.Object):
+    @bus.Object.method(in_types=['s'])
+    def start(self, _bridge):
+        ...
```

Comment:
Hmm, why doesn't this make the pycheck scenario burst up in flames? This file is full of `...`

Reply:
`<class 'ellipsis'>` what on earth is that now? Sometimes it feels like Python is getting worse..

Reply:
`...` is a conventional alternative for "I'll implement this later"

It's basically equivalent to a docstring, or `pass`.